src/Dapr.Actors/Client/ActorProxyOptions.cs

```diff
@@ -26,5 +30,24 @@ public JsonSerializerOptions JsonSerializerOptions
             set => this.jsonSerializerOptions = value ??
                     throw new ArgumentNullException(nameof(JsonSerializerOptions), $"{nameof(ActorProxyOptions)}.{nameof(JsonSerializerOptions)} cannot be null");
         }
+
+        /// <summary>
+        /// The Dapr Api Token that is added to the header for all requests.
+        /// </summary>>
+        public string DaprApiToken
+        {
+            get => this.daprApiToken;
+            set => this.daprApiToken = value;
+        }
```

Comment:
you can just use `{ get; set; }` and there's no need for an explicit backing field

Reply:
I would expect this to have its value initialized by the env-var